File: statistics/factorial.ts
/**
 * [Factorial 阶乘]
 * @param  {number} n [阶乘数]
 * @return {number}   [阶乘值]
 */
function Factorial(n:number):number{
		let total:number=1,k:number=n%1;
		for(let i=1;i<=n;i++){
			total*=i;
		}
		return total;
	}
export {Factorial}

File: statistics/C/src/factorial.c
#include "factorial.h"
/**
 * [Factorial 阶乘]
 * @param  n [description]
 * @return   [description]
 */
long int Factorial(int n){
    long int total=1;
    for(int i=1;i<=n;i++){
        total*=i;
    }
    return total;
}
